etc/modules.d/gpu.yaml

```diff
@@ -5,52 +5,11 @@ env: ENABLE_GPU
 additional_args:
   - -e NVIDIA_VISIBLE_DEVICES
 copy:
-  - ../01-gpu.conf:/etc/ld.so.conf.d/01-gpu.conf
```

Comment:
Maybe it makes more sense to remove `gpu` and `mpich` modules from this repo all together?